content/1e/classes/cleric.mdx

```diff
@@ -90,6 +90,16 @@ abilities:
       Immediately choose a nearby ally and give them the blessing as a free action.
 
       An ally with this blessing can use it to reroll an attack as a free action this battle. An ally can only have one such blessing on them at a time.
+      
+      **Invocation of Justice/Vengeance:** This battle, add double your level to the miss damage of your attacks and the attacks of your
+nearby allies. (For example, your basic melee attack as a cleric will deal triple your level as miss damage while this invocation is active.)
```

Comment:
```suggestion
      nearby allies. (For example, your basic melee attack as a cleric will deal triple your level as miss damage while this invocation is active.)
```

Just needs to be re-aligned since YAML is very picky like that :smile_cat:

Reply:
Actually I'll go ahead and make the fix since github lets me do that easily